newdemojail.yaml

```diff
@@ -0,0 +1,772 @@
+    name: "reported2"
+  reported3:
+    fqn: "ol.reported"
+    src: "inmate"
+    dst: "sexoffenderstatus"
+    entitySetName: "DemoReported"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        transforms:
+          - !<transforms.ConcatTransform>
+            columns: ["SubjectIdentification","SexOffender"]
+    name: "reported3"
+  isregisteredsexoffenderin:
+    fqn: "ol.isregisteredsexoffenderin"
+    src: "inmate"
+    dst: "county"
+    entitySetName: "DemoIsregisteredsexoffenderin"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+          - !<transforms.ConcatTransform>
+            columns: ["SubjectIdentification","SexOffenderCounty"]
+    name: "isregisteredsexoffenderin"
+  registeredin:
+    fqn: "ol.registeredin"
+    src: "sexoffenderstatus"
+    dst: "county"
+    entitySetName: "DemoRegisteredin"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+          - !<transforms.ConcatTransform>
+            columns: ["SexOffender","SexOffenderCounty"]
+    name: "registeredin"
+  resultsin:
+    fqn: "ol.resultsin"
+    src: "sentence"
+    dst: "jailstay"
+    entitySetName: "DemoResultsin"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+          - !<transforms.ConcatTransform>
+            columns: ["SentenceId","BookId"]
+    name: "resultsin"
+  oflength2:
+    fqn: "ol.oflength"
+    src: "jailstay"
+    dst: "jailstaylength"
+    entitySetName: "DemoLengthof"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+          - !<transforms.ConcatTransform>
+            columns: ["BookId","JailstayLength"]
+    name: "oflength2"
+  partof:
+    fqn: "ol.partof"
+    src: "hearing"
+    dst: "courtcase"
+    entitySetName: "DemoPartof"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "DocketNo"
+    name: "partof"
+  partof2:
+    fqn: "ol.partof"
+    src: "jailstay"
+    dst: "courtcase"
+    entitySetName: "DemoPartof"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+          - !<transforms.ConcatTransform>
+            columns: ["BookId","DocketNo"]
+    name: "partof2"
+  observedin:
+    fqn: "ol.observedin"
+    src: "mentalhealthstatus"
+    dst: "inmate"
+    entitySetName: "DemoObservedin"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+          - !<transforms.ConcatTransform>
+            columns: ["MentalHealthStatus","SubjectIdentification"]
+    name: "observedin"
+  occurredduring:
+    fqn: "ol.occurredduring"
+    src: "mentalhealthstatus"
+    dst: "jailstay"
+    entitySetName: "DemoOccurredduring"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+          - !<transforms.ConcatTransform>
+            columns: ["MentalHealthStatus","BookId"]
```

Comment:
please add a datetime property to this association. Can be the jail start date.

Reply:
Added